api/app/reviews/api.py

```diff
@@ -294,6 +332,28 @@ def get(self):
 
         return ReviewResponseUser(review_form, response, 0, args['language'], review_response=review_response)
 
+class ResponseReviewEventAdminAPI(restful.Resource):
+    @event_admin_required
+    def get(self, event_id):
+        parser = reqparse.RequestParser()
+        parser.add_argument('response_id', type=int, required=True)
+        parser.add_argument('language', type=str, required=True)
+        args = parser.parse_args()
+
+        response_id = args['response_id']
+
+        review_form = review_repository.get_review_form(event_id)

```

Comment:
So there can actually be multiple review forms for an event representing multi-stage review processes (see `stage` parameter on the review repository). So what you're doing is showing the reviews for the active review stage, which is fine, but maybe just put a note on the UI under your "Reviews" heading saying that this only shows the active stage (and in fact you can get the stage number from the review form and display that too). Don't worry about adding support for multiple stages now, instead please raise a new issue to add support for this and we can do that later if someone actually needs it.